recipe/patch_yaml/mamba.yaml

```diff
@@ -129,3 +129,15 @@ then:
   - tighten_depends:
       name: conda
       upper_bound: "23.9.0"
+---
+# conda=23.11.0 broke mamba's cli
+# https://github.com/conda/conda/issues/13400
+if:
+  name: mamba
+  version_ge: 1.5.2
+  version_le: 1.5.3
+  timestamp_lt: 1701773340000
+then:
+  - replace_depends:
+      old: conda >=23.9,<24
```

Comment:
For the record, this permissive pin is what made this compatibility issue a lot more pressing than it should be.